Yoast/yoast-components
# Introduce wrapper in mock to pass innerRef as a ref and re-enable tests

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Introduced a wrapper in the mock for ReplacementVariableEditorStandalone which passes `innerRef` as a `ref`. ReplacementVariableEditorStandalone needs the ref prop because the `focus` function needs it.
* Re-enable tests that are dependent on this mock.

## Test instructions

This PR can be tested by following these steps:

* Run `yarn test` and check that all tests pass.
